cg/meta/workflow/nf_analysis.py

```diff
@@ -521,6 +521,7 @@ def run_analysis(
 
     def get_deliverables_template_content(self) -> list[dict[str, str]]:
         """Return deliverables file template content."""
+        LOG.info("Get deliverables file template content")
```

Comment:
```suggestion
        LOG.debug("Getting deliverables file template content")
```

Let's not crowd the logs, I would remove or set as debug